tensorflow/g3doc/tutorials/mnist/pros/index.md

```diff
@@ -157,11 +157,12 @@ easily. Our cost function will be the cross-entropy between the target and the
 model's prediction.
 
 ```python
-cross_entropy = -tf.reduce_sum(y_*tf.log(y))
+cross_entropy = tf.reduce_mean(-tf.reduce_sum(y_ * tf.log(y), reduction_indices=[1]))
 ```
 
-Note that `tf.reduce_sum` sums across all images in the minibatch, as well as
-all classes. We are computing the cross entropy for the entire minibatch.
+Note that `tf.reduce_sum` sums across all classes and `tf.reduce_mean` takes 
+the average over these sums. We are computing the cross entropy for the entire
```

Comment:
Delete the last sentence "We are computing the cross entropy for the entire minibatch."

(It was a bit of an odd phrasing -- but it was intended to convey that we were computing the cross entropy as if a sample consisted of an entire minibatch -- and this is no longer true.)